app/src/main/java/com/gdfp/android_evaluationv2/MainActivity.java

```diff
@@ -32,6 +32,7 @@
     private static final String TAG = "MainActivity";
 
     private TaskAdapter mAdapter;
+    private SQLiteDatabase mDatabase;
```

Comment:
I don't think it should be here because al the transactions must be in the corresponding service, if this not the case please explain your scenario